modules/apps/data-engine/data-engine-test/src/testFunctional/tests/macros/DataEngineUpgrade.macro

```diff
@@ -2,9 +2,33 @@ definition {
 
 	macro createNewStructureUsingAnUpgradedStructureAsFieldset {
 
-		// TODO LPS-126232 createNewStructureUsingAnUpgradedStructureAsFieldset pending implementation
+		WebContentNavigator.openWebContentAdmin(siteURLKey = "test-site-name");
```

Comment:
Hey @DiegoDOC , you cannot use this test-site-name since we cannot assume pre-upgraded WEM structures were created on this group. I've updated to the guest user for safety. Thanks.